src/main/java/com/google/firebase/messaging/AndroidNotification.java

```diff
@@ -602,6 +617,19 @@ public Builder setNotificationCount(int notificationCount) {
       return this;
     }
 
+    /**
+     * Sets the proxy of this notification.
+     *
+     * @param proxy The proxy value. one of the values in
+     * {PROXY_UNSPECIFIED, ALLOW, DENY, IF_PRIORITY_LOWERED}
```

Comment:
Let update the docs here as well, Thanks!
```suggestion
     * @param proxy The proxy value, one of the values in {ALLOW, DENY, IF_PRIORITY_LOWERED}
```